webapp/utils/utils.jsx

```diff
@@ -165,6 +165,10 @@ export function ding() {
     }
 }
 
+export function hasNoSoundOptions() {
```

Comment:
Thanks @Whiteaj36 for your contribution! Looks good to me except for minor item. I suggest to name this function as `hasSoundOptions` and subsequently update its logic implementation.